### DISCUSSION TITLE
how to output search result all on one line

### DISCUSSION THREAD

**Discussion by misfitTOY** [Homebrew/discussions#4606] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.26-2-gc5cbe64
ORIGIN: https://github.com/Homebrew/brew
HEAD: c5cbe6406b8b9d040c5d1850b5f041af4ddef83c
Last commit: 8 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 192e51edd731d9b062b6afdaf5623bd65cbcd1ee
Core tap last commit: 5 months ago
Core tap branch: master
Core tap JSON: 27 Jun 23:53 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.R6ZrJowAQi/org.xquartz:0
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.41.0 => /opt/homebrew/bin/git
Curl: 7.88.1 => /usr/bin/curl
macOS: 13.4.1-arm64
CLT: N/A
Xcode: 14.3.1
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

i'm sorry to have to ask, but i've honestly been searching for an answer to this simple question: 

when one does a search, the results are always listed on separate lines, unless there is a lot, then it will put 3,4,5 on the same line - basically makes columns.. how does one get the search results to format into one line?

example:
brew search font

and the results will look something like this:
font-sksdkd
font-akdkdk
font-alsdlkd
etc.

how do you make the results format like this:
font-sksdkd font-akdkdk font-alsdlkd etc.

thanks in advance - i know it's simple, i just can't find it..

**Comment by osalbahr** [comment, marked as answer]
try this maybe?
```bash
$ brew search font | xargs echo
```

**Reply by misfitTOY** [reply]
omg - thank you so so much!! - it was driving me crazy! thank you so much!

![some-old-guy](https://github.com/Homebrew/discussions/assets/90716990/c428cfaf-ab5c-46dd-bf63-b6439224ce3b)

**Reply by osalbahr** [reply]
You're welcome! :)
